### ISSUE TITLE
Multiple Nesting Layers

### ISSUE THREAD

**Issue by louisefindlay23** [beaverbuilder/feature-requests#20]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I can't easily create the layout I want because I can't have multiple nesting layers.

**Describe the solution you'd like**
Add support for BB plugin for multiple nesting layers.

**Additional context**
See this [GIF](https://screenshots.louisefindlay.com/BluO0GJ5) for an example of the problem.

**Comment by zackpyle** [comment]
Agreed. Sometimes I would like to have more nested columns than is currently allowed
